bulkS3upload.go

```diff
@@ -69,20 +69,27 @@ var errorLines = 0
 // and then waits on a channel for file paths that should be copied into the endpoint/bucket
 func copyWorker(bucket string, url string, accessID string, secretKey string, ssl bool, files <-chan string, nodeStats chan<- CopyResult, wg *sync.WaitGroup) {
 
+	ctx := context.Background()
```

Comment:
The new MinIO call requires a context (and this is just the one example code used).